CHANGELOG.md

```diff
@@ -1,5 +1,59 @@
+* [Teleport (preview)](https://github.com/Azure/acr/blob/main/docs/teleport/aks-getting-started.md) on AKS will be retired on 15 July 2025, please [migrate to Artifact Streaming (preview) on AKS](https://learn.microsoft.com/azure/aks/artifact-streaming) or update your node pools to set --aks-custom-headers EnableACRTeleport=false. Azure Container Registry has removed the Teleport API meaning that any nodes with Teleport enabled are pulling images from Azure Container Registry as any other AKS node. After 15 July 2025, any node pools with Teleport (preview) enabled may experience breakage and node provisioning failures. For more information, see [aka.ms/aks/teleport-retirement](https://aka.ms/aks/teleport-retirement).
+
+### Release Notes
+* Features  
+  * Kubernetes 1.31 and 1.32 are now designated as [Long-Term Support (LTS)](https://learn.microsoft.com/azure/aks/long-term-support) versions.  
+  * [Kubernetes 1.33](https://kubernetes.io/blog/2025/04/23/kubernetes-v1-33-release/) is available in Preview. A full matrix of supported add-ons and components is published at the [AKS versions page](https://learn.microsoft.com/azure/aks/supported-kubernetes-versions?tabs=azure-cli#kubernetes-1330).
+  * [Network-isolated clusters]( https://learn.microsoft.com/azure/aks/concepts-network-isolated) are now Generally Available.
+  * GA support for [CNIv1 → Cilium and Cilium → Overlay](https://learn.microsoft.com/azure/aks/upgrade-azure-cni) upgrade paths.
+
+* Bug Fixes  
+  * Fixed failures triggered by duplicate [tag keys](https://learn.microsoft.com/azure/aks/use-tags) that differed only by character case.
+
+* Behavior Changes  
+  * [AKS `default` maintenance window](https://learn.microsoft.com/azure/aks/planned-maintenance?tabs=azure-cli)now ignores maintenance windows that aren’t weekly.   
+  * Updated the Windows GPU device-plugin to `v0.0.19`, mitigating [CVE-2025-22871](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2025-22871).
+  * Bumped [Container Insights image](https://github.com/microsoft/Docker-Provider/blob/ci_prod/ReleaseNotes.md) and agent versions to 3.1.27 for both Linux and Windows agents.
+  * Static egress gateway memory limits raised for greater stability. 
+* Component Updates  
+  * Updated [Node Auto Provisioning]( https://learn.microsoft.com/azure/aks/node-autoprovision?tabs=azure-cli) Custom Resource Definitions to [`1.4.0-aks`](https://github.com/Azure/karpenter-provider-azure/blob/main/charts/karpenter-crd/templates/karpenter.azure.com_aksnodeclasses.yaml)
+    * Added zone-label support and image-status fields.  
+    * Introduced `v1beta1` API for AKSNodeClass; default CRs updated accordingly.  
+    * Deployment flow now consumes `vnetGUID` from Secret when present.  
+    * Clean-up of obsolete resource versions/labels; broader ConfigMap read permissions for maintenance-window detection.  
```

Comment:
There are multiple small updates here which I see also called out under https://github.com/Azure/karpenter-provider-azure/releases/tag/v1.4.0. For example v1beta1 API for AKSNodeClass. Any special call outs about those? or should we just reference the 1.4.0 release note directly? on the 'broder ConfigMap read permissions for maintenance window detection' line, we typically call out permissions changes under behavior change section. can we move it there? and on that, can we mention which identity got the additional permissions and what were the additional permissions itself?